examples/technical_blogs/tartan_blog/build_and_run_networks.py

```diff
@@ -0,0 +1,156 @@
+import subprocess
+import os
+import shutil
+import requests
+
+# Prereqs: portBLAS, SYCL-DNN, oneapi oneapi_construction_kit should be cloned and installed.
+portBLAS_source = os.environ.get("portBLAS_source")
+SYCL_DNN_source = os.environ.get("SYCL-DNN_source")
+path_to_oneapi = os.environ.get("path_to_oneapi")
+OCK_libCL_path = os.environ.get("OCK_libCL_path")
+input_image = os.environ.get("input_image")
+
+# Set environment variables
+os.environ["CMAKE_CXX_COMPILER"] = os.path.join(path_to_oneapi, "compiler/linux/bin-llvm/clang++")
```

Comment:
I dunno if we want to be more generic and avoid path separators everywhere, e.g., `os.path.join(path_to_oneapi, "compiler", "linux", "bin-llvm", "clang++")`? I notice you do that sort of thing further down in this script.

I don't mind, as long as we're consistent. It doesn't look like we're going to be running this on Windows anyway.